src/encoded/schemas/changelogs/library.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for library.json
 
+### Minor changes since schema version 9
+
+* *queried_RNP_size_range* property is added, the property captures ribonucleoprotein size range for eCLIP controls.
+
```

Comment:
it is indeed ribonucleoprotein size range, but not limited to eCLIP controls only - it could be specified in any eCLIP (and potentially other assays). Please omit the specification of eCLIP control. It can be something along the lines: "the property captures ribonucleoprotein size range targeted   in the assay, the library was constructed for."